src/Proto3/Suite/Tutorial.hs

```diff
@@ -92,21 +93,25 @@ instance Named Bar
 -- |
 -- == Enumerations
 --
--- Enumerated types can be used by deriving the 'Enum', 'Finite' and 'Named'
--- classes. Each of these instances are implied by a 'Generic' instance, so can
--- be derived as follows:
+-- Enumerated types can be used by deriving the 'Bounded', 'Enum', 'ProtoEnum',
+-- 'Finite', and 'Named' classes.  Each of these instances are implied by
+-- a 'Generic' instance, so can be derived as follows:
 --
 -- > data Shape
 -- >   = Circle
 -- >   | Square
 -- >   | Triangle
 -- >   deriving (Generic, Enum, Finite, Named)
+-- >
+-- > instance ProtoEnum Shape
```

Comment:
If you use `DeriveAnyClass` you can add `ProtoEnum` to the `deriving` clause.  In fact, it appears that the derived instances for `Finite` and `Named` are already assuming the use of `DeriveAnyClass` because they would not work otherwise

Reply:
@Gabriel439 , thanks; fixed by 2deefae0e25583ba203772a5c8ea6f08953549be .